### PR TITLE
#3735 Add tooltip storybook

### PR DESCRIPTION
**Pull request by PavelVanecek**
## Description

Added MDX and storybook

## Related Issue

https://github.com/recharts/recharts/issues/3735

## Motivation and Context

Documentation I guess

## How Has This Been Tested?

Eslint and playing around in the browser

## Screenshots (if appropriate):

image

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [X] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [X] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [X] All new and existing tests passed.

### CODE REVIEW

File: storybook/stories/API/component/Tooltip.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,274 @@
+import React from 'react';
+import { ResponsiveContainer, Tooltip, LineChart, Line } from '../../../../src';
+import { pageData } from '../../data';
+import { getStoryArgsFromArgsTypesObject } from '../props/utils';
+
+type MyArgType = {
```
Comment (ckifer) [inline storybook/stories/API/component/Tooltip.stories.tsx:6]: nit: can we make this a common type (maybe `RechartsArgType`) that extends `ArgType` from storybook?

**Review by ckifer — APPROVED** [review]
LGTM, one small comment about the type there but not a huge deal

File: storybook/stories/API/component/Tooltip.stories.tsx (hunk 1 above)
Comment (PavelVanecek) [reply storybook/stories/API/component/Tooltip.stories.tsx:6]: Yep I'll do that in a followup PR.

Comment (PavelVanecek) [reply storybook/stories/API/component/Tooltip.stories.tsx:6]: https://github.com/recharts/recharts/pull/3761

**Pull request merged**